CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # moi changelog
 
-## Version 0.1.0-dev, changes go here
+## Version 0.1.0-dev
+
+Currently in development, released on May 2nd 1023
```

Comment:
hah... oops, ya, had that in when prodding the tool for testing. not
convinced it fully works yet, will resolve last kinks when releasing 0.2

On Tue, Jun 2, 2015 at 12:59 PM, adamrp <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/biocore/mustached-octo-ironman/pull/34#discussion_r31558401>
> :
>
> > @@ -1,6 +1,8 @@
> >  # moi changelog
> >
> > -## Version 0.1.0-dev, changes go here
> > +## Version 0.1.0-dev
> > +
> > +Currently in development, released on May 2nd 1023
>
> Perhaps the most important event that year besides The Ghaznavid Empire's
> occupation of Transoxiana <http://en.wikipedia.org/wiki/1023#Asia>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/mustached-octo-ironman/pull/34/files#r31558401>
> .
>